examples/BasicOperations/AddCampaigns.cs

```diff
@@ -120,7 +120,9 @@ public void Run(GoogleAdsClient client, long customerId)
                     {
                         TargetGoogleSearch = true,
                         TargetSearchNetwork = true,
-                        TargetContentNetwork = false,
+                        // Enable Display Expansion on Search campaigns. See
+                        // https://support.google.com/google-ads/answer/7193800 to learn more.
+                        TargetContentNetwork = true,
```

Comment:
Is enabling this by default now considered best practice?

Reply:
This is all the discussion from b/183989977. Happy to sync offline.